x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc

```diff
@@ -31,25 +32,52 @@ The following parameters can be specified in the body of a DELETE request and
 pertain to invalidating tokens:
 
 `token` (optional)::
-(string) An access token. This parameter cannot be used any of `refresh_token`, `realm_name` or
-         `username` are used.
+(string) An access token. This parameter cannot be used any of `refresh_token`,
+`realm_name` or `username` are used.
 
 `refresh_token` (optional)::
-(string) A refresh token. This parameter cannot be used any of `refresh_token`, `realm_name` or
-                          `username` are used.
+(string) A refresh token. This parameter cannot be used any of `refresh_token`,
+`realm_name` or `username` are used.
 
 `realm_name` (optional)::
-(string) The name of an authentication realm. This parameter cannot be used with either `refresh_token` or `token`.
+(string) The name of an authentication realm. This parameter cannot be used with
+either `refresh_token` or `token`.
 
 `username` (optional)::
-(string) The username of a user. This parameter cannot be used with either `refresh_token` or `token`
+(string) The username of a user. This parameter cannot be used with either
+`refresh_token` or `token`
 
-NOTE: While all parameters are optional, at least one of them is required. More specifically, either one of `token`
-or `refresh_token` parameters is required. If none of these two are specified, then `realm_name` and/or `username`
-need to be specified.
+NOTE: While all parameters are optional, at least one of them is required. More
+specifically, either one of `token` or `refresh_token` parameters is required.
+If none of these two are specified, then `realm_name` and/or `username` need to
+be specified.
 
 ==== Examples
 
+For example, if you create the following token:
+
+[source,js]
+--------------------------------------------------
+POST /_security/oauth2/token
+{
+  "grant_type" : "client_credentials"
+}
+--------------------------------------------------
+// CONSOLE
+
+The API returns the following information about the access token:
+
+[source,js]
+--------------------------------------------------
+{
+  "access_token" : "dGhpcyBpcyBub3QgYSByZWFsIHRva2VuIGJ1dCBpdCBpcyBvbmx5IHRlc3QgZGF0YS4gZG8gbm90IHRyeSB0byByZWFkIHRva2VuIQ==",
+  "type" : "Bearer",
+  "expires_in" : 1200
+}
+--------------------------------------------------
+// TESTRESPONSE[s/dGhpcyBpcyBub3QgYSByZWFsIHRva2VuIGJ1dCBpdCBpcyBvbmx5IHRlc3QgZGF0YS4gZG8gbm90IHRyeSB0byByZWFkIHRva2VuIQ==/$body.access_token/]
+
+
```

Comment:
Suggestion: 

This access token can now be immediately invalidated, as shown in the following example: